src/docs/content/observability/observability-scenario/pre-requisites/oci-logging.md

```diff
@@ -0,0 +1,106 @@
+---
```

Comment:
This pre-req should be optional. In many settings users may not have the capacity and makes the "getting started" curve steep.

Reply:
idea was to have users understand active and passive monitoring (looking at OCI monitoring dashboard and getting alerts via alarms.) Plus looking at logs to root cause the issue.
How can OCI observability help determine failure to observe to action. This was the intent. I agree its a bit of a curve.